packages/cli/src/bin.ts

```diff
@@ -5,6 +5,7 @@ export const run = async () => {
   const cli = build()
     .brand("webstone")
     .src(__dirname)
+    .plugins("./node_modules", { matching: "webstone-plugin-*" })
     .plugins("./node_modules", { matching: "webstone-plugin-*-cli" })
```

Comment:
Do you think we should keep this for backwards-compatibility?

Reply:
I don't think so. The only plugin we've released to NPM so far is [webstone-plugin-request-logger-web](https://www.npmjs.com/package/webstone-plugin-request-logger-web). It does not include CLI commands and I'll deprecate it once the new structure is done and we're happy with it.